### PR TITLE
UI cleanup

### PR DESCRIPTION
**Pull request by VictoriaKGN**
- user is now redirected to greetings page if they try to access an insider page without being logged in
- removed `isLoggedIn` prop from header as the header is used only when logged in

### CODE REVIEW

File: client/src/App.tsx
Comment (dlarocque) [inline client/src/App.tsx:55]: Remove commented code

Comment (VictoriaKGN) [reply client/src/App.tsx:55]: I will uncomment before merging. this branch does not have the Expenses page yet. will merge once Expenses is merged

**Review by dlarocque — APPROVED** [review]

**Pull request merged**
